diff/diff.go

```diff
@@ -64,8 +66,63 @@ func Manifests(oldIndex, newIndex map[string]*manifest.MappingResult, options *O
 		doDiff(&report, key, nil, newContent, options)
 	}
 
+	filteredReport := Report{}
```

Comment:
It isn't strictly needed, but if you have some time, it would be nicer to extract everything into a dedicated function like:

```golang
report, err := doSuppress(report, options.SuppressedOutputLineRegex)
if err != nil {
    panic(err)
}

seenAnyChanges := len(report.entries) > 0
report.clean()
return seenAnyChanges
```

This way, we can hide `filteredReport` behind the doSuppress function and it can be unit-tested easily. WDYT?

Reply:
Good call, I hope this fine for now.